CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+6.3.1
+=====
+
+* fix #625: restore tomli in install_requires after the regression changes in took it out
+  and some users never added it even tho they have pyproject.toml files
```

Comment:
I don't frequent various Python lists, so I don't understand why I suddenly have to add "tomli" as a dependency in my pyproject.toml...

Reply:
wit the change in here you dont, initially one was supposed to use `setuptools_scm[toml]`

Reply:
I'm using `setuptools_scm[toml]` [everywhere here]( https://github.com/scikit-hep/cookie/search?q=setuptools_scm) and it's still failing, not sure why.

Reply:
(Could be my mistake still somehow, but does show it's tricky to get right)

Reply:
I think this is transitive, and black is the package breaking. Why it's triggering this at all and not just using a wheel, I don't know.

Reply:
Ah, okay, so the problem was that tomli was required even when not using toml configuration, so that's what was breaking black. (Again, still don't know why this wasn't getting a wheel and skipping this entirely, but that's why)